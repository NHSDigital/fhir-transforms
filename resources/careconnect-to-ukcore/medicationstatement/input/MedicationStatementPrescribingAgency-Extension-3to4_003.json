{
  "resourceType": "MedicationStatement",
  "extension": [
    {
      "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-MedicationPrescribingAgency-1",
      "valueCodeableConcept": {
        "coding": {
          "code": "prescribed-at-gp-practice",
          "display": "Prescribed at GP practice"
        }
      }
    }
  ]
}
