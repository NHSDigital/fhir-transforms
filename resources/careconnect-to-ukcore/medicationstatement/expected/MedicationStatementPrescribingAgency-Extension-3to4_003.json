{
  "resourceType": "MedicationStatement",
  "extension": [
    {
      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-MedicationPrescribingOrganization",
      "valueCodeableConcept": {
        "coding": [
          {
            "code": "prescribed-at-gp-practice",
            "display": "Prescribed at GP practice"
          }
        ]
      }
    }
  ]
}